### ISSUE TITLE
[Defold] Builder fails while bob is resolving dependencies

### ISSUE THREAD

**Issue by britzl** [herman-rogers/GambuildrRoadmap#15]
When building a project with dependencies the log from bob.jar shows "java.io.IOException: Server returned HTTP response code: 400 for URL: https://github.com/britzl/ludobits/archive/1.2.zip" The URL is valid so the question is if there's something else amiss?

**Comment by herman-rogers** [comment]
Is this the URL you've used inside the buildr?

**Comment by britzl** [comment]
No, the URL is a dependency in game.project of the main project. This line to be specific: https://github.com/britzl/oneroom/blob/master/game.project#L4

So the Buildr manages to get the main project and tries via bob.jar to resolve the project dependencies, but for some reason fails to download the dependency. Is the Buildr somehow restricted from downloading files while building?

**Comment by herman-rogers** [comment]
It shouldn't be restricted from downloading but something could be happening in the build process. If you don't mind, I may grab these files and try the build myself and see if I can get some debugging information out of it.
